robolectric/src/test/java/org/robolectric/shadows/ShadowCharArrayBufferTest.java

```diff
@@ -13,8 +13,8 @@
   public void append_shouldBeInstrumented() {
     // CharArrayBuffer.expand uses a modified version of java.lang.System.arraycopy.
     // The class needs to be instrumented and re-written in order to work correctly.
-    final CharArrayBuffer buffer = new CharArrayBuffer(1);
-    buffer.append("Foo");
-    assertThat(buffer.toString()).isEqualTo("Foo");
+    // final CharArrayBuffer buffer = new CharArrayBuffer(1);
```

Comment:
Not sure sorry.

On 11 November 2015 at 07:28, Tom Rudick <notifications@github.com> wrote:

> In
> robolectric/src/test/java/org/robolectric/shadows/ShadowCharArrayBufferTest.java
> <https://github.com/robolectric/robolectric/pull/2137#discussion_r44544355>
> :
>
> > @@ -13,8 +13,8 @@
> >    public void append_shouldBeInstrumented() {
> >      // CharArrayBuffer.expand uses a modified version of java.lang.System.arraycopy.
> >      // The class needs to be instrumented and re-written in order to work correctly.
> > -    final CharArrayBuffer buffer = new CharArrayBuffer(1);
> > -    buffer.append("Foo");
> > -    assertThat(buffer.toString()).isEqualTo("Foo");
> > +    // final CharArrayBuffer buffer = new CharArrayBuffer(1);
>
> Will do. Although, I think this is the only test testing System.arrayCopy
> is instrumented. Do you know of another place where arrayCopy is used that
> I could modify this test to use?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robolectric/robolectric/pull/2137/files#r44544355>.
>